pkgs/by-name/ux/uxn/package.nix

```diff
@@ -7,7 +7,7 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "uxn";
-  version = "unstable-2024-02-15";
+  version = "0-unstable-2024-02-15";
```

Comment:
1. This was unrelated to the PR & commit title.
2. Just slapping a `0` in there for the sake of being guideline-conformant is wrong. Correct version would've been `1.0-unstable-2024-02-15`.

Reply:
Sorry.